rtos/Queue.h

```diff
@@ -107,6 +107,17 @@ class Queue : private mbed::NonCopyable<Queue<T, queue_sz> > {
         return osMessageQueueGetSpace(_id) == 0;
     }
 
+    /** Check the quantity of items in queue.
```

Comment:
It is not really a check. To be more precise, you could write: 
"Get number of queued messages in the queue"
and
@return Number of items in the queue